### PR TITLE
Fix logo size

### PR DESCRIPTION
**Pull request by jxltom**
Continue https://github.com/mirumee/saleor/pull/3696.

I found the logo size in firefox and chrome have different sizes. So using specific size such as ```113px``` is better than ```100%``` or ```auto``` width. This PR fixes the logo size in top, footer and checkout.

This PR also fixes the navigation menu in mobile has too much top padding issue.

### Screenshots

Before:
saleor e commerce-3
screenshot from 2019-02-14 14-55-24

After:

saleor e commerce-2
screenshot from 2019-02-14 14-50-15

### Pull Request Checklist

1. [x] Privileged views and APIs are guarded by proper permission checks.
1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [x] Database migration files are up to date.
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] GraphQL schema and type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

**Comment by michaljelonek** [comment]
That hamburger menu icon on mobile looks weirdly placed or is it just me? I feel it should be centered, but it's more to the top. Or is it that way by design? @maarcingebala 

Very professional screenshot: 
image

**Comment by jxltom** [comment]
Same for me. :P

I feel that the menu icon should be centered vertially. I've tried some simple css but with no luck. Please feel free to make any improvements.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Unify field names on product, collection and page - #3706 by @michaljelonek
 - Generate voucher code if it wasn't provided in mutation - #3717 by @Kwaidan00
 - Reuse Storefront's 1.0 payment logic in API - #3715 by @maarcingebala
+- Fix logo size in different browser and devices with different sizes -3722 # by @jxltom
```
Comment (Pacu2) [inline CHANGELOG.md]: I think we are tagging issues differently

Comment (jxltom) [reply CHANGELOG.md]: > I think we are tagging issues differently

Thanks! Fixed!

**Comment by maarcingebala** [comment]
As @michaljelonek noticed we could align the menu icon better. @dominik-zeglen any ideas on how we could do that?

**Comment by dominik-zeglen** [comment]
We could try flexbox

**Comment by jxltom** [comment]
@dominik-zeglen Thanks for the hints, I found simply setting svg width can also solve this issue.
@maarcingebala Now the PR is updated, please have a look.

**Comment by jxltom** [comment]
And here is comparison of dashboard logo:

Before:
screenshot from 2019-02-26 11-57-09
screenshot from 2019-02-26 11-57-22

After:
screenshot from 2019-02-26 11-56-03
screenshot from 2019-02-26 11-56-35

**Pull request merged**